build.gradle.kts

```
@@ -27,7 +27,7 @@ val currentOs = org.gradle.internal.os.OperatingSystem.current()
 
 kotlin {
 
-    val bitcoinKmpVersion = "0.16.0" // when upgrading bitcoin-kmp, keep secpJniJvmVersion in sync!
+    val bitcoinKmpVersion = "0.17.0-SNAPSHOT" // when upgrading bitcoin-kmp, keep secpJniJvmVersion in sync!
```

```suggestion
    val bitcoinKmpVersion = "0.17.0" // when upgrading bitcoin-kmp, keep secpJniJvmVersion in sync!
```

Needs a rebase, and we should take the opportunity to also switch to the new `Chain` type (https://github.com/ACINQ/bitcoin-kmp/pull/117).